src/views/DocumentCompare.vue

```diff
@@ -76,6 +77,20 @@ export default {
 			documents: undefined
 		};
 	},
+	computed: {
+		backToPrev: function () {
+			if (prevRoute.name === 'graph') return { path: prevRoute.fullPath };
```

Comment:
if preVroute is undefined (we haven't cached the route (means it came from elsewhere, return matches) else return preVroute, fullPath?